test/lib/lbt/analyzer/JSModuleAnalyzer.js

```diff
@@ -287,8 +287,6 @@ test.cb("OldStyleBundleV2", (t) => {
 	);
 });
 
-// Note: this test still fails as the evo-style bundles don't declare the names of the
-// contained raw-modules any longer.
```

Comment:
Was this comment removed intentionally?

Reply:
I guess, yes. Because this TODO was, what #341 wanted to addressed (migration of 3562114).

Reply:
I thought this fix made the tests not fail anymore therefore I removed it